in.varb.Ausweiskopie.yaml

```diff
@@ -0,0 +1,75 @@
+app-id: "in.varb.Ausweiskopie"
```

Comment:
https://varb.in/ is 404 not found. You have to put something there so that I know you own it. A link to github etc.

Reply:
Ah, right. Alternatively, is https://links.varb.in enough for a verification, or is verification with a subdomain not acceptable?

Reply:
Github link should be https://github.com/varbin not https://github.com/@varbin

anyway that's enough.

Reply:
Oh, thanks for the finding.